HTML/public/css/style.css

```diff
@@ -0,0 +1,24 @@
+@font-face {
+  font-family: "Arcade Classic";
+  src: url("/HTML/public/assets/font/Arcade-Classic.TTF");
+}
+@font-face {
+  font-family: "VP Pixel";
+  src: url("/HTML/public/assets/font/VP-Pixel.otf");
+}
+
+.title h1 {
+  font-family: "Arcade Classic";
+}
+
+h1 {
+  font-family: "VP Pixel";
+}
+
+p {
+  font-family: "VP Pixel";
+}
+
+.button {
+  font-family: "VP Pixel";
+}
```

Comment:
this is better to remove it and you only just use class that you've setup on `tailwind.config`

Reply:
jadi karena metodenya masih CDN itu ga bisa di custom dl bang, pas nanti di jadiin blade baru aku ganti ke CLI biar bisa custom config

Reply:
aku make CDN supaya nanti ga bentrok pas push sama yg ngerjain html yang lain

Reply:
ooh gitu toh okeyy thankyou penjelasannya